users/views.py

```diff
@@ -0,0 +1,10 @@
+from . import models, serializers
+from balarco import utils
+
+
+class UserViewSet(utils.GenericViewSet):
+    """ViewSet for User CRUD REST Service that inherits from utils.GenericViewSet
+    """
+    obj_class = models.User
```

Comment:
Shouldn't you be using permission classes so that only certain users can query their information and that only admin users can create users?